web/wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -0,0 +1,151 @@
+<?php
+// If on Pantheon
+if( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) ){
+	// Disable WordPress auto updates
+	if( ! defined('WP_AUTO_UPDATE_CORE')) {
+		define( 'WP_AUTO_UPDATE_CORE', false );
+	}
+
+	remove_action( 'wp_maybe_auto_update', 'wp_maybe_auto_update' );
+	// Remove the default WordPress core update nag
+    add_action('admin_menu','_pantheon_hide_update_nag');
+}
+
+function _pantheon_hide_update_nag() {
+	remove_action( 'admin_notices', 'update_nag', 3 );
+	remove_action( 'network_admin_notices', 'update_nag', 3 );
+}
+
+// Get the latest WordPress version
+function _pantheon_get_latest_wordpress_version() {
+	$core_updates = get_core_updates( array('dismissed' => false) );
+
+	if( ! is_array($core_updates) || empty($core_updates) || ! property_exists($core_updates[0], 'current' ) ){
+		return null;
+	}
+
+	return $core_updates[0]->current;
+}
+
+// Check if WordPress core is at the latest version.
+function _pantheon_is_wordpress_core_latest() {
+	$latest_wp_version = _pantheon_get_latest_wordpress_version();
+
+	if( null === $latest_wp_version ){
+		return true;
+	}
+
+	// include an unmodified $wp_version
+	include( ABSPATH . WPINC . '/version.php' );
+
+	// Return true if our version is the latest
+	return version_compare( str_replace( '-src', '', $latest_wp_version ), str_replace( '-src', '', $wp_version ), '=' );
+
+}
+
+// Return upstream's org and repository names, or false if not a custom upstream
+// (i.e. Pantheon WordPress or wordpress-network )
```

Comment:
:no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).